robots/robot.js

```diff
@@ -1,9 +1,30 @@
 function Robot(startCharge) {
+  this._battery = startCharge
+  this._position = [0, 0]
+  this._history = []
+}
 
+Robot.prototype.battery = function() {
+  return this._battery
 }
 
 Robot.prototype.position = function() {
+  return this._position
+}
+
+
+Robot.prototype.moveLeft = function(x) {
+  this._history.push('left')
+  return this._position[0] = this._position[0] - 1
+}
+
+Robot.prototype.moveUp = function(x) {
+  this._history.push('up')
+  return this._position[1] = this._position[1] + 1
```

Comment:
Don't return on a setter `=` - it always evaluates to nothing.
In order to return something useful, you might add a line like this: `return this._position[1]`